client/get_transitions.go

```diff
@@ -0,0 +1,32 @@
+package client
+
+import (
+	"errors"
+	"fmt"
+	"net/http"
+)
+
+type TransitionsResult struct {
+	Transitions []TransitionObj `json:"transitions"`
+}
+type TransitionObj struct {
+	TransitionId   string `json:"id"`
+	TransitionName string `json:"name"`
+}
+
+func GetTransitions(issueId string) (TransitionsResult, error) {
+	result := TransitionsResult{Transitions: []TransitionObj{}}
+	path := fmt.Sprintf("/rest/api/2/issue/%s/transitions", issueId)
+	request, err := constructGetRequest(path)
+	if err != nil {
+		return result, err
```

Comment:
This is super tiny thing, but are the indents supposed to be that big? 8 spaces is what it looks like.

Reply:
Good catch Rod - I guess @exp-dev-dj is also commenting the same thing... It's from the IDE itself. I will see how to edit them. Thank you!!